admin/console/hidden-settings.rst

```diff
@@ -79,7 +97,6 @@ Sample of the above setting:
 
 .. image:: /images/console/ui_ticket_create_notes.gif
 
-
 Show E-Mail-Address of customer on customer selection (Ticket-Creation)
```

Comment:
"Email address" instead of "E-Mail-Address". "ticket creation" instead of "Ticket-Creation".